_data/navigation.yml

```diff
@@ -1,4 +1,6 @@
-- title: Competencia
+- title: Inicio
+  url: acmucsp
```

Comment:
Use the direct url: https://ucsp.acm.org/, and change the title to `ACM UCSP`.

Reply:
I put "Inicio" because main page. I tried seem the same.

Reply:
It is a reference to a page outside the wiki, the name should be more explicit.

Reply:
do it